### ISSUE TITLE
Is this plugin not compatible with osTicket v1.18.1? Nothing works

### ISSUE THREAD

**Issue by hongz1** [philbertphotos/osticket-multildap-auth-plugin#83]
I recently installed osTicket 1.18.1 and this plugin. It has 400 not found error when I select lookup found user in manual user add function in User Directory.

LDAP sync does not work either...

**Comment by philbertphotos** [comment]
I am creating a fresh install of 1.18.1 and will do a few tests to fix this issue.

**Comment by hongz1** [comment]
Please set a enough field length to the filter so that I can add my custom filter.

**Comment by hongz1** [comment]
When is the revision one coming?

**Comment by philbertphotos** [comment]
@hongz1 its now compatible with 1.18.1 and changed the logic for quite a few things. It should be release on or before Sunday.

**Comment by hongz1** [comment]
Thanks for the good news. Hopefully I can install it on Monday.

**Comment by philbertphotos** [comment]
@hongz1 it will only work with a single instance in this FIX just but all the domains in ONE instance. This is due to a bug in osticket I believe. I almost finished an it will be released shortly

**Comment by hongz1** [comment]
Does ldap sync works? This is what I need now

**Comment by philbertphotos** [comment]
Yes found the bug that stopped it from working and know i know why it mostly worked for just me. It was an issue with the new instance feature that i never took into account on a fresh install. Sent from my iPhoneOn Jul 28, 2024, at 2:11 PM, Jae Hong Lee ***@***.***> wrote:﻿
Does ldap sync works? This is what I need now

**Comment by philbertphotos** [comment]
Ok its been uploaded .. looking forward to hearing good things.
Fresh install works and all attributes seem to be doing what it supposed today.

**Comment by hongz1** [comment]
Ok. I will try reinstall it and let you know whether it works. Thanks.

**Comment by hongz1** [comment]
I redownloaded and installed. still not working on everywhere. does not show lookup list at all and does not work with ldap sync.
auth-ldap plugin worked perfectly though. where can I see any error message? I turned on debug mode on, but was not able to find any error messages. What I saw was that web browser console shows errors when I navigate to multi-ldap conf page.

**Comment by hongz1** [comment]
This plugin also messed up staff login page. When I login with admin user (manually created), it keeps spinning with Verifying message. when I refresh page, it goes to backend page. Something is definitely bugged out.

**Comment by hongz1** [comment]
check connection button doesn't do anything.

**Comment by hongz1** [comment]
![image](https://github.com/user-attachments/assets/4490f742-f9a9-41e6-bd07-2b261679afa7)

**Comment by philbertphotos** [comment]
@hongz1 verified and fixing all of this ... to be honest. I never tried to login into staff because I never touched that part.
Fixed the Check Connection button I had some of it disabled.
So let me fix those last bits test that and LDAP search and update the plugin. 

Also do verbose DEBUG it shows more stuff.

**Comment by hongz1** [comment]
Login staff issue is nothing related to its page. It is caused by syntax error on your config form page.

Thank you for all your work. Please let me know when it's done.

**Comment by hongz1** [comment]
Any updates?

**Comment by philbertphotos** [comment]
Had to work backward to find why … found the bug … just going over auth.php a bit more before i release the other files work as is. Sent from my iPhoneOn Aug 1, 2024, at 12:10 PM, Jae Hong Lee ***@***.***> wrote:﻿
Any updates?

**Comment by philbertphotos** [comment]
@hongz1 I hope I killed it all ...the bugs .
I tested it over and over and over.

Have some cleaning up to do but so far so good.
Let me know I am working on it all weekend so a fix will be relatively quick.

**Comment by philbertphotos** [comment]
![image](https://github.com/user-attachments/assets/3a77c148-6bc7-4dec-9b87-a98b56fc67e7)

**Comment by philbertphotos** [comment]
![image](https://github.com/user-attachments/assets/6de2ebdd-e43e-4a26-b8e4-c7fe4b007b3b)

**Comment by philbertphotos** [comment]
Closed already fixed. plugin updated for the latest version
